index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Assignment3</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="stylesheet.css" >
+    </head>
+    <body>
+        <div class="container">
+            <div class="row">
+                <div class="col1">
```

Comment:
![image](https://user-images.githubusercontent.com/92850233/141088453-d0d6c6ab-f91e-4127-9eeb-c2683ccd3a56.png)
optional https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Styling_tables
use tables it can be easier , less styling required

Reply:
Yeah, I was doing so to ma
![](https://user-images.githubusercontent.com/92850233/141088453-d0d6c6ab-f91e-4127-9eeb-c2683ccd3a56.png)
ke myself more comfortable with CSS.